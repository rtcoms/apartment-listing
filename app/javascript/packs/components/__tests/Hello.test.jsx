import Hello from '../Hello.jsx'

import React from 'react';
import renderer from 'react-test-renderer';

test('hello should render with name', () => {
  const component = renderer.create(
    <Hello name="Kasaz">Facebook</Hello>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
