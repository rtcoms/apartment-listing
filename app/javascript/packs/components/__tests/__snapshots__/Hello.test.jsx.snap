// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hello should render with name 1`] = `
<div>
  Hello 
  Kasaz
  !
</div>
`;
